WindowsServerDocs/administration/server-core/what-is-server-core.md

```diff
@@ -49,42 +49,61 @@ The following table shows which applications are available *locally* on Server C
+| Eventvwr                           | with FOD 1903   | available                      |
 | Wevtutil (Event queries)           | available       | available                      |
 | Services.msc                       | not available   | available                      |
 | Control Panel                      | not available   | available                      |
 | Windows Update (GUI)               | not available   | available                      |
-| Windows Explorer                   | not available   | available                      |
+| Windows Explorer                   | with FOD 1903   | available                      |
 | Taskbar                            | not available   | available                      |
 | Taskbar notifications              | not available   | available                      |
 | Taskmgr                            | available       | available                      |
-| Internet Explorer or Edge          | not available   | available                      |
+| Taskscheduler (taskschd.msc)       | with FOD 1903   | available                      |
+| Internet Explorer                  | with FOD 1903   | available                      |
+| Edge                               | not available   | available**                    |
+| Edge Chromium (Enterprise)         | optional install| optional install               |
 | Built-in help system               | not available   | available                      |
 | Windows 10 Shell                   | not available   | available                      |
 | Windows Media Player               | not available   | available                      |
 | PowerShell                         | available       | available                      |
-| PowerShell ISE                     | not available   | available                      |
+| PowerShell ISE                     | FOD 1903***     | available***                   |
 | PowerShell IME                     | available       | available                      |
 | Mstsc.exe                          | not available   | available                      |
 | Remote Desktop Services            | available       | available                      |
-| Hyper-V Manager                    | not available   | available                      |
+| Hyper-V Manager                    | FOD 1903        | available                      |
 | WordPad\*                          | not available   | available                      |
 
+\* To read .RTF  files locally stored on a Server Core SKU, users can copy the file(s) to a different Windows computer where WordPad is present.
+
+Features indicated with 2 asterisks ( \** ) are end of support by March 21, 2021. Edge Chromium is a more versatile and modern Browser to replace Internet Explorer and EdgeHTML.
+
+Features indicated with 3 asterisks ( \*** ) are officially deprecated. Please consider to use Visual Studio Code instead of ISE.
```

Comment:
Minor phrasing detail:
```suggestion
Features indicated with 3 asterisks ( \*** ) are officially deprecated. Please consider using Visual Studio Code instead of ISE.
```